<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="runners.GetUser" tests="0" skipped="0" failures="0" errors="0" timestamp="1970-01-01T00:00:00" hostname="QV-4398" time="0.0">
  <properties/>
  <system-out><![CDATA[Feature from file:///C:/Automatizaciones/Proyectos_Automatizacion/BDDPracticeServices/src/test/resources/features/get_user.feature

?[31mFailed scenarios:?[0m
?[31mfile:///C:/Automatizaciones/Proyectos_Automatizacion/BDDPracticeServices/src/test/resources/features/get_user.feature:13?[0m# Get response

1 Scenarios (?[31m1 failed?[0m)
2 Steps (?[31m1 failed?[0m, ?[32m1 passed?[0m)
0m5,858s


io.cucumber.core.exception.CucumberException: Step [I can see the dates of the user] is defined with 0 parameters at 'stepsdefinitions.GetUserStepDefinition.iCanSeeTheDatesOfTheUser()'.
However, the gherkin step has 1 arguments:
 * Table:
      | First Name | Janet  |
      | Last Name  | Weaver |
      | Id         | 2      |

Step text: I can see the dates of the user
	at io.cucumber.core.runner.PickleStepDefinitionMatch.arityMismatch(PickleStepDefinitionMatch.java:134)
	at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:42)
	at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:64)
	at io.cucumber.core.runner.TestStep.run(TestStep.java:49)
	at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:46)
	at io.cucumber.core.runner.TestCase.run(TestCase.java:51)
	at io.cucumber.core.runner.Runner.runPickle(Runner.java:66)
	at io.cucumber.junit.PickleRunners$NoStepDescriptions.run(PickleRunners.java:149)
	at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:83)
	at io.cucumber.junit.FeatureRunner.runChild(FeatureRunner.java:24)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:227)
	at io.cucumber.junit.CucumberSerenityRunner.runChild(CucumberSerenityRunner.java:54)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at io.cucumber.junit.CucumberSerenityRunner$RunCucumber.evaluate(CucumberSerenityRunner.java:254)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:119)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:182)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:164)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:414)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
	at java.lang.Thread.run(Thread.java:748)


]]></system-out>
  <system-err><![CDATA[[Test worker] WARN io.cucumber.junit.CucumberSerenityRunner - By default Cucumber is running in --non-strict mode.
This default will change to --strict and --non-strict will be removed.
You can use --strict or @CucumberOptions(strict = true) to suppress this warning
[Test worker] INFO  - 

-------------------------------------------------------------------------------------
     _______. _______ .______       _______ .__   __.  __  .___________.____    ____ 
    /       ||   ____||   _  \     |   ____||  \ |  | |  | |           |\   \  /   / 
   |   (----`|  |__   |  |_)  |    |  |__   |   \|  | |  | `---|  |----` \   \/   /  
    \   \    |   __|  |      /     |   __|  |  . `  | |  |     |  |       \_    _/   
.----)   |   |  |____ |  |\  \----.|  |____ |  |\   | |  |     |  |         |  |     
|_______/    |_______|| _| `._____||_______||__| \__| |__|     |__|         |__|    
                                                                                     
 News and tutorials at http://www.serenity-bdd.info                                  
 Documentation at https://wakaleo.gitbooks.io/the-serenity-book/content/             
 Join the Serenity Community on Gitter: https://gitter.im/serenity-bdd/serenity-core 
 Serenity BDD Support and Training at http://serenity-bdd.info/#/trainingandsupport  
 Learn Serenity BDD online at https://www.serenitydojo.net                           
-------------------------------------------------------------------------------------

[Test worker] INFO  - Test Suite Started: As a user I want get information with relation to clients
[pool-2-thread-1] WARN net.thucydides.core.model.failures.FailureAnalysisConfiguration - Could not find error class: exceptions(exceptions)
java.lang.ClassNotFoundException: exceptions
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.typesDefinedIn(FailureAnalysisConfiguration.java:146)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.compromisedTypesDefinedIn(FailureAnalysisConfiguration.java:129)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.pendingTypes(FailureAnalysisConfiguration.java:85)
	at net.thucydides.core.model.failures.FailureAnalysis.reportAsPending(FailureAnalysis.java:111)
	at net.thucydides.core.model.failures.FailureAnalysis.resultFor(FailureAnalysis.java:44)
	at net.thucydides.core.model.TestOutcome.testResultFromFailureClassname(TestOutcome.java:1361)
	at net.thucydides.core.model.TestOutcome.getResult(TestOutcome.java:1349)
	at net.thucydides.core.model.TestOutcome.calculateDynamicFieldValues(TestOutcome.java:347)
	at net.thucydides.core.reports.json.gson.GsonJSONConverter.toJson(GsonJSONConverter.java:81)
	at net.thucydides.core.reports.json.JSONTestOutcomeReporter.generateReportFor(JSONTestOutcomeReporter.java:60)
	at net.thucydides.core.reports.ReportService.generateReportFor(ReportService.java:239)
	at net.thucydides.core.reports.ReportService.lambda$generateReportsFor$0(ReportService.java:177)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[pool-2-thread-1] WARN net.thucydides.core.model.failures.FailureAnalysisConfiguration - Could not find error class: exceptions(exceptions)
java.lang.ClassNotFoundException: exceptions
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.typesDefinedIn(FailureAnalysisConfiguration.java:146)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.compromisedTypesDefinedIn(FailureAnalysisConfiguration.java:129)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.skippedTypes(FailureAnalysisConfiguration.java:97)
	at net.thucydides.core.model.failures.FailureAnalysis.reportAsSkipped(FailureAnalysis.java:123)
	at net.thucydides.core.model.failures.FailureAnalysis.resultFor(FailureAnalysis.java:47)
	at net.thucydides.core.model.TestOutcome.testResultFromFailureClassname(TestOutcome.java:1361)
	at net.thucydides.core.model.TestOutcome.getResult(TestOutcome.java:1349)
	at net.thucydides.core.model.TestOutcome.calculateDynamicFieldValues(TestOutcome.java:347)
	at net.thucydides.core.reports.json.gson.GsonJSONConverter.toJson(GsonJSONConverter.java:81)
	at net.thucydides.core.reports.json.JSONTestOutcomeReporter.generateReportFor(JSONTestOutcomeReporter.java:60)
	at net.thucydides.core.reports.ReportService.generateReportFor(ReportService.java:239)
	at net.thucydides.core.reports.ReportService.lambda$generateReportsFor$0(ReportService.java:177)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[pool-2-thread-1] WARN net.thucydides.core.model.failures.FailureAnalysisConfiguration - Could not find error class: exceptions(exceptions)
java.lang.ClassNotFoundException: exceptions
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.typesDefinedIn(FailureAnalysisConfiguration.java:146)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.compromisedTypesDefinedIn(FailureAnalysisConfiguration.java:129)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.compromisedTypes(FailureAnalysisConfiguration.java:70)
	at net.thucydides.core.model.failures.FailureAnalysis.reportAsCompromised(FailureAnalysis.java:99)
	at net.thucydides.core.model.failures.FailureAnalysis.resultFor(FailureAnalysis.java:50)
	at net.thucydides.core.model.TestOutcome.testResultFromFailureClassname(TestOutcome.java:1361)
	at net.thucydides.core.model.TestOutcome.getResult(TestOutcome.java:1349)
	at net.thucydides.core.model.TestOutcome.calculateDynamicFieldValues(TestOutcome.java:347)
	at net.thucydides.core.reports.json.gson.GsonJSONConverter.toJson(GsonJSONConverter.java:81)
	at net.thucydides.core.reports.json.JSONTestOutcomeReporter.generateReportFor(JSONTestOutcomeReporter.java:60)
	at net.thucydides.core.reports.ReportService.generateReportFor(ReportService.java:239)
	at net.thucydides.core.reports.ReportService.lambda$generateReportsFor$0(ReportService.java:177)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[pool-2-thread-1] WARN net.thucydides.core.model.failures.FailureAnalysisConfiguration - Could not find error class: exceptions(exceptions)
java.lang.ClassNotFoundException: exceptions
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.typesDefinedIn(FailureAnalysisConfiguration.java:146)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.compromisedTypesDefinedIn(FailureAnalysisConfiguration.java:129)
	at net.thucydides.core.model.failures.FailureAnalysisConfiguration.failureTypes(FailureAnalysisConfiguration.java:61)
	at net.thucydides.core.model.failures.FailureAnalysis.reportAsFailure(FailureAnalysis.java:87)
	at net.thucydides.core.model.failures.FailureAnalysis.resultFor(FailureAnalysis.java:53)
	at net.thucydides.core.model.TestOutcome.testResultFromFailureClassname(TestOutcome.java:1361)
	at net.thucydides.core.model.TestOutcome.getResult(TestOutcome.java:1349)
	at net.thucydides.core.model.TestOutcome.calculateDynamicFieldValues(TestOutcome.java:347)
	at net.thucydides.core.reports.json.gson.GsonJSONConverter.toJson(GsonJSONConverter.java:81)
	at net.thucydides.core.reports.json.JSONTestOutcomeReporter.generateReportFor(JSONTestOutcomeReporter.java:60)
	at net.thucydides.core.reports.ReportService.generateReportFor(ReportService.java:239)
	at net.thucydides.core.reports.ReportService.lambda$generateReportsFor$0(ReportService.java:177)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]></system-err>
</testsuite>
